src/lib.rs

```diff
@@ -110,6 +110,15 @@ impl TodoList {
     pub fn get_task(&mut self) -> Option<Todo> {
         self.items.pop()
     }
+
+    /// Show all tasks
+    pub fn show_tasks(&mut self) {
+        //self.items.sort_unstable()
+        println!("Here are all of your tasks");
+        for item in self.items {
```

Comment:
```suggestion
        for item in &self.items {
```
make the borrow checker happy